content/en/platform/corda/4.11/community/get-started/tutorials/supplementary-tutorials/supplementary-tutorials-intro.md

```diff
@@ -25,6 +25,6 @@ As these tutorials don’t depend on the code from any of the other tutorials, y
 | **Oracles**      | [Writing oracle services](oracles.html)                                                                                                                                                                                                                                            |
```

Comment:
Add relref to these ones?

Reply:
I skipped these ones for the moment as I wanted to find out was there a reason for non-MD links with no anchors in a table - I thought it might be an edge case. Will revisit.